spec/ffi/function_type_spec.rb

```diff
@@ -0,0 +1,27 @@
+#
+# This file is part of ruby-ffi.
+# For licensing, see LICENSE.SPECS
+#
+
+require File.expand_path(File.join(File.dirname(__FILE__), "spec_helper"))
+
+describe FFI::FunctionType, skip: RUBY_PLATFORM == "java" do
```

Comment:
The `java` ext doesn't define this class, I suppose that's why it wasn't tested. But I think these specs are worth it for the MRI implementation.

Reply:
Yes, FunctionInfo is just an internal class not exposed through the API.